db/migrate/20230404170721_create_comments.rb

```diff
@@ -1,7 +1,7 @@
 class CreateComments < ActiveRecord::Migration[7.0]
   def change
     create_table :comments do |t|
-      t.references :author, foreign_key: { to_table: 'users' }
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ] nice you are very close but you should remove the `s` from `users` so the foreign_key will be `user_id` as in the diagram not `users_id`

Reply:
Thank you for the heads up, sir 👍 

I've gone ahead and **also** renamed the `posts_id` column to `post_id`, along with the `users_id` to `user_id` as per instruction and the ERD diagram.

![image](https://user-images.githubusercontent.com/11477535/229954162-517d8a3b-8f8a-4bd4-b8b9-24da02882dda.png)

Please note that the ERD diagram depicts the `user_id` in both the likes and comments tables as "AuthorID."
I made the switch to `user_id` from `author_id` in these tables as a result of the first code review.

![image](https://user-images.githubusercontent.com/11477535/229954794-05fd7325-4ac5-484c-9032-c7c11d3f8f37.png)